tests/resource_test.py

```diff
@@ -17,3 +18,22 @@ def test_static_files():
         "myapp-schema not in list={}",
         x,
     )
+
+
+def test_render_resource():
+    from sirepo import resource
+    from pykern import pkunit
+
+    d = pkunit.data_dir()
+    pkunit.file_eq(
+        d.join("expect.py"),
```

Comment:
I think the fact that these are Python files confuses things. Having them be text files is better, because there's no implied semantics for someone reading this in the "tests as documentation" mode.